src/sentry/projectoptions/defaults.py

```diff
@@ -39,7 +39,7 @@
 # The JavaScript loader version that is the project default.  This option
 # is expected to be never set but the epoch defaults are used if no
 # version is set on a project's DSN.
-register(key="sentry:default_loader_version", epoch_defaults={1: "4.x", 2: "5.x"})
+register(key="sentry:default_loader_version", epoch_defaults={1: "4.x", 2: "5.x", 3: "6.x"})
```

Comment:
```suggestion
register(key="sentry:default_loader_version", epoch_defaults={1: "4.x", 2: "5.x", 7: "6.x"})
```

Reply:
OK, thank you. You were right @rhcarvalho